packages/docs/package.json

```diff
@@ -35,10 +35,9 @@
     "gatsby": "^2.26.1",
     "gatsby-plugin-catch-links": "^2.4.0",
     "gatsby-plugin-compile-es6-packages": "^2.1.0",
-    "gatsby-plugin-mdx": "^2.0.1",
+    "gatsby-plugin-mdx": "^1",
```

Comment:
Quite a rollback, I have very little experience with Gatsby so cannot say if it looks good or bad, but just checking if that was intentional

Reply:
Thanks mate! Good catch. I downgraded it to see if the bump to v2 was the cause of the problem, as I had problems with gatsby-plugin-mdx 2.5.x (similar to https://github.com/gatsbyjs/gatsby/issues/31392, but on Linux not Windows). Turns out downgrading to 2.0.1 is enough.